lib/splash/track_er_screen.dart

```diff
@@ -11,7 +13,44 @@ class TrackErScreen extends StatelessWidget {
       appBar: AppBar(
         iconTheme: const IconThemeData(color: primaryColor),
       ),
-      body: MostlyRoundedButton(buttonText: 'Next'),
+      body: Center(
+        child: FractionallySizedBox(
+          widthFactor: 0.9,
+          child: Column(
+            children: [
+              const Spacer(flex: 2),
+              Image.asset('assets/ER_Group.png'),
+              const Spacer(flex: 1),
+              const Align(
+                alignment: Alignment.centerLeft,
+                child: Text("Track ER Progress", style: TextStyle(fontSize: 26, fontWeight: FontWeight.w600))
+              ),
+              const Align(
+                alignment: Alignment.centerLeft,
+                child: Text("Welcome to our cutting edge platform where you can effortlessly be informed about you or your loved ones care in the Emergency Department. We provide real time updates to your care so you can be at ease knowing the clinical staff are here to take great care of you.", 
```

Comment:
I prefer to break these strings into more viewable content. In the middle of the string you can hit return and it then displays on two lines in code. ie 'hello ' 'world' == 'hello world'